src/main/java/org/openmainframeproject/cobolcheck/features/interpreter/InterpreterController.java

```diff
@@ -349,6 +350,19 @@ else if(reader.isFlagSet(Constants.LEVEL_01_TOKEN)){
                 lineRepository.addFileSectionStatement(line.getUnNumberedString());
             }
         }
+        if(reader.isFlagSet(Constants.WORKING_STORAGE_SECTION) && 
+        line.containsToken(Constants.EXEC_SQL_TOKEN) && 
+        (line.containsToken(Constants.INCLUDE) || reader.appendNextMeaningfulLineToCurrentLine().containsToken(Constants.INCLUDE))) {
```

Comment:
Use `peekNextMeaningfulLine` instead of appending. No need to alter the source code if it is not needed - that is, if we end up appending something that is not `INCLUDE`. If you need it to be appended, you can do that inside the if-statement.